cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -278,6 +280,26 @@ share
     txId                TxId
     UniqueTreasury      addrId txId
 
+  -- -----------------------------------------------------------------------------------------------
+  -- Multi Asset related tables.
+
+  MaPolicy
+    hash                ByteString          sqltype=hash28type
+    UniqueMAPolicy      hash
+
+  MultiAsset
+    policyId            MaPolicyId
+    name                ByteString          sqltype=asset32type
+    quantity            DbWord64            sqltype=word64type
+    txId                TxId
+    UniqueMa            policyId txId name
+
+  MaTxOut
+    assetId             MultiAssetId
+    quantity            DbWord64            sqltype=word64type
+    txOutId             TxOutId
+    UniqueMaTxOut       assetId txOutId
```

Comment:
Name bike shedding:

The names "policy id" and "asset id" already have meanings in the MA spec, so it would be very confusing to people trying to write queries to have those names refer to other things (i.e. the artificial primary keys).

Reply:
"id" here is short for "index". This is the same naming scheme that is used throughout the rest of the database schema.

And "id" is what Persistent calls it.